apps/heft/src/plugins/SassTypingsPlugin/SassTypingsGenerator.ts

```diff
@@ -110,6 +119,10 @@ export class SassTypingsGenerator extends StringValuesTypingsGenerator {
     });
   }
 
+  private _isSassPartial(filePath: string): boolean {
+    return path.basename(filePath)[0] === '_';
```

Comment:
Maybe throw a comment in here that this is always expected to be the case, to clarify why we do this to future devs seeing the code

Reply:
@D4N14L Hello Daniel, I think I've addressed all your feedback.